samples/nrf9160/modem_shell/src/location/CMakeLists.txt

```diff
@@ -8,3 +8,10 @@ target_include_directories(app PRIVATE .)
 
 target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/location_shell.c)
 target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/location_cmd_utils.c)
+
+if(CONFIG_LOCATION_SERVICE_EXTERNAL)
```

Comment:
I would put these ext stuff to own subdir under location or at least have ext in filenames.

Reply:
There's just 4 c-files (and another 4 others) in this directory so I'm not sure we need a separate directory for clarity for 3/8 files.

I'm thinking if the files would be `location_srv_ext.h` etc. (irrespective of where they are). Or `location_service_ext.h`.
I'm tempted to go for shorter name (`location_srv_ext.h`).

Comments?

Reply:
Your proposal with _ext_ is ok for me.

Reply:
This is done now.